packages/react-pose/src/posed.tsx

```diff
@@ -28,22 +29,34 @@ const componentCache = new Map<
 
 const createComponentFactory = (key: string | React.ComponentType) => {
   const componentFactory: ComponentFactory<any> = (poseConfig = {}) => {
-    return ({ withParent = true, ...props }) => {
+    // TODO: Replace functional context with new class property API
+    return forwardRef(({ withParent = true, ...props }, ref) => {
+      warning(
```

Comment:
if u want to keep compat for v4 u need to pass `ref || props.innerRef` as `innerRef` to `PoseElement`

Reply:
This should be provided by ...props, overwriting innerRef={ref} if present

Reply:
Shouldnt ref win over innerRef prop though?

Reply:
Yeah but I can’t imagine a situation in which they’ll both be defined